frontend/src/components/CreateProjectDialog.tsx

```diff
@@ -1,94 +1,103 @@
-import React, { useState } from 'react';
-import '../styles/CreateProjectDialog.css';
+import React, { useState } from "react";
+import "../styles/CreateProjectDialog.css";
 
 // Define an enum for error types
 enum ErrorType {
-    None = '',
-    ProjectCreationFailed = 'Failed to create project. Please try again.',
-    InvalidInput = 'Invalid input. Please check your entries.',
+    None = "",
+    ProjectCreationFailed = "Failed to create project. Please try again.",
+    InvalidInput = "Invalid input. Please check your entries.",
+    Success = "Project created successfully!",
 }
 
-
 export default CreateProjectDialog;
 function CreateProjectDialog() {
-    const[isDialogOpen, setIsDialogOpen] = useState(false);
-    const [projectName, setProjectName] = useState('');
-    const [description, setDescription] = useState('');
-    const [projectID, setProjectID] = useState('');
+    const [isDialogOpen, setIsDialogOpen] = useState(false);
+    const [projectName, setProjectName] = useState("");
+    const [description, setDescription] = useState("");
+    const [projectID, setProjectID] = useState("");
     const [error, setError] = useState<ErrorType>(ErrorType.None);
 
-    const openDialog = () =>setIsDialogOpen(true);
-    const closeDialog = ()=> setIsDialogOpen(false);
-
+    const openDialog = () => setIsDialogOpen(true);
+    const closeDialog = () => {
+        setIsDialogOpen(false);
+        setError(ErrorType.None);
+    };
 
     const handleSubmit = async (e: React.FormEvent) => {
```

Comment:
this doesn't need to be an async function. you can just resolve the promise directly

Reply:
I tried but I kept failing the Quality Gate check

Reply:
Also if I am using 'await', I thought it needs to be an async function.

Reply:
You can use Promise.then() and friends

Reply:
Got it I made the change.